File: URI/Cpp/1165.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
  int n = 0, x = 0;

  cin >> n;
  while (n--) {
    cin >> x;

    bool primo = true;
    for (int i = 2; i < x; i++) {
      if(x % i == 0) {
        primo = false;
        break;
      }
    }

    cout << x << (primo ? " eh primo" : " nao eh primo") << endl;
  }
   
  return 0;
}

File: URI/python/1165.py

# https://www.urionlinejudge.com.br/judge/pt/problems/view/1165

import math

n = int(input())

for _ in range(n):
  x = int(input())
  raiz = int(math.sqrt(x))

  primo = True
  if (x % 2 != 0):
    for i in range(3, raiz + 1, 2):
      if(x % i == 0):
        primo = False
        break
  elif(x != 2):
    primo = False
  
  print(x, 'eh primo' if primo else 'nao eh primo')

